drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_priv.h

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
What made you consider making this a data a separate header file?
Nit But what do you think about moving this into the common.c?

Reply:
it's not clear to me what you are commenting about because your comment is just at the top of the file

Reply:
I mean moving the entire file into common.c

Reply:
I'm not sure what you mean, the header files has definitions that the other 3 files need